AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -157,6 +157,21 @@ public async Task<IActionResult> CloneHearing(Guid hearingId, MultiHearingReques
             }
         }
 
+        private static List<DateTime> GetDatesForClonedHearings(MultiHearingRequest hearingRequest)
+        {
+            if (hearingRequest.HearingDates != null && hearingRequest.HearingDates.Any())
```

Comment:
Is this to cover the scenario when VHOs provide a list of single dates?

Reply:
yes that's correct - HearingDates contains the individual list of dates